### ISSUE TITLE
Allow impeti to display both a Reveal message and a mishap message

### ISSUE THREAD

**Issue by Robotgiggle** [gamma-delta/HexMod#1168]
### Describe the feature

Using Reveal in a spell circle places the revealed iota into the scry-tooltip on the impetus. However, it seems to use the same slot in that tooltip as mishap messages, so if the circle mishaps after revealing something then the mishap message will override the Reveal message. This behavior makes it very hard to use Reveal for circle debugging, since the error you're trying to fix overwrites the debug info you're using to fix it.

Ideally, the scry-tooltip should be able to display both a Reveal message _and_ a mishap message at the same time.
